webhooks/finished/public/connect.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <title>Connect!</title>
+</head>
+
+<body class="text-center">
+  <div class="container w-75 pt-5">
+    <h3>Let's connect to your bank!</h3>
+    <button type="button" class="btn btn-primary btn-md opacity-50 mt-3" id="startLink">Connect my bank</button>
+
+  </div>
+  <script src="https://cdn.plaid.com/link/v2/stable/link-initialize.js"></script>
+  <script src="js/connect.js"></script>
+</body>
+
+</html>
```

Comment:
question: just curious why this is a separate html.  could we just make the "connect your bank" link in the index.html page a button to launch link?

Reply:
Hmmm... yeah, we could. I think this is kind of a carryover from the OAuth tutorial, where it was easier to move all the connect logic into a separate page. Lemme try combining everything back into a single page and see how that looks...

Reply:
Okay, done. I changed the scripts into modules to support the importing of functions in a simple way -- are there any gotchas there I should be aware of?

Reply:
not that i know of.